eas.json

```diff
@@ -4,7 +4,7 @@
   },
   "build": {
     "production": {
-      "node": "16.13.0",
+      "node": "v18.19.1",
```

Comment:
is it fine to have the version `v` letter added here now?

Reply:
I missed this. There's not supposed to be a `v`!

https://docs.expo.dev/build/eas-json/#selecting-build-tool-versions

<img width="281" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/50935f47-397f-425f-8d8a-85e1f6b53235">

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1136/commits/c577e0b584718f35904aa20681b54f8c7bbc35ae